libs/error-stack/src/fmt/color.rs

```diff
@@ -161,7 +161,7 @@ impl Foreground {
 
         let buffer = &[if bright { b'9' } else { b'3' }, color.digit()];
         // This should never fail because both are valid ASCII
-        let control = core::str::from_utf8(buffer).unwrap();
+        let control = core::str::from_utf8(buffer).expect("valid utf-8 buffer");
```

Comment:
Same as above, the output of `expect` is different

Reply:
fixed in [c96800b](https://github.com/hashintel/hash/pull/2075/commits/c96800bc492e75dfa78f7c665d009bf691af73b4)